### ISSUE TITLE
Request: Groups

### ISSUE THREAD

**Issue by mrbiber** [PeerioTechnologies/peerio-client#23]
It would be great if one could create groups of contacts to share files with

**Comment by Leodagan** [comment]
Do you try to create a conversation with multiple recipients then send them a file ?

**Comment by httng** [comment]
The request is about being able to define that selection of multiple recipients as a group. Basically: Labeling them.

**Comment by hdf** [comment]
+1 for "labeling" contacts. :)

**Comment by mrbiber** [comment]
Yes, exactly, labeling. If you send files / messages to the same group of people multiple times a day, this saves you a great deal of typing.

**Comment by kaepora** [comment]
I'm changing the title of this ticket to "Create contact labels/groups."

This is an interesting feature and something we should implement sooner rather than later. Thanks for the good suggestion.

**Comment by praxis-journal** [comment]
This feature would be very useful. Group creation would allow users to create e2e encrypted mailing lists/listservs as well as small collaborative groups. Group creation and mass distribution are critical to activist work. Being able to easily define multiple overlapping working groups is also key for collaborative productivity.

**Comment by skylarnagao** [comment]
Default group: "recent contacts"

Include all added or messaged contacts from the past 48 hours.
